test/operations.py

```diff
@@ -0,0 +1,94 @@
+from http import HTTPStatus
+import requests
+import json
+import os
+
+
+class Operations:
+    @staticmethod
+    def create_index(endpoint, index_name, auth=None, data=None):
```

Comment:
I don't see the `auth` and `data` arguments being used; is that something you're going to add later?

Also - I recommend you add type hints to your Python code.